development/imagebump-detector/pkg/common.go

```diff
@@ -0,0 +1,49 @@
+package common
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"os"
+	"regexp"
+
+	"gopkg.in/yaml.v3"
+)
+
+func ParseNotationFile(filePath string) (string, string, error) {
+	f, err := os.Open(filePath)
+	defer f.Close()
+	if err != nil {
+		panic(err)
+	}
+	r := bufio.NewReader(f)
+	for {
+		line, err2 := r.ReadString('\n')
+		if err2 == io.EOF {
+			break
+		} else if err != nil {
+			fmt.Print(fmt.Errorf("error: %s", err2))
```

Comment:
If the `err` is not nil then the read file is unusable. You should return the error here.
```suggestion
return "", "", err
```